### ISSUE TITLE
The result of training is very poor.

### ISSUE THREAD

**Issue by hahahaa1** [mkang315/RCS-YOLO#11]
According to the requirements in the paper, under the conditions of batch_size = 8 and epochs = 150, the result trained in the train.py file is very poor. There is no default training weight file in the train.py file. I don't know if this is the reason why my training result is very unsatisfactory. Could the author please answer me?
![微信图片_20240506135833](https://github.com/mkang315/RCS-YOLO/assets/156444982/b951aa63-2510-4ae4-a09e-2994a135f334)
![微信图片_20240506135905](https://github.com/mkang315/RCS-YOLO/assets/156444982/787388b8-d270-46ff-9c50-c115431e39d4)

**Comment by mkang315** [comment]
Precisely, the training result is meager. I cannot judge what's wrong with your settings only by the screenshot.

**Comment by hahahaa1** [comment]
I am reproducing on the Windows system. In the train.py file, I have made the following settings: batch_size = 8 and epochs = 150, but the result is very unsatisfactory. I would like to ask if you used pre-trained weights when conducting the training. My training result is really a headache. Thank you for your answer.
![Uploading 微信图片_20240506192005.png…]()
